types/validation.go

```diff
@@ -57,8 +57,14 @@ func VerifyCommit(chainID string, vals *ValidatorSet, blockID BlockID,
 //
 // This method is primarily used by the light client and does not check all the
 // signatures.
-func VerifyCommitLight(chainID string, vals *ValidatorSet, blockID BlockID,
-	height int64, commit *Commit) error {
+func VerifyCommitLight(
+	chainID string,
+	vals *ValidatorSet,
+	blockID BlockID,
+	height int64,
+	commit *Commit,
+	countAllSignatures bool,
```

Comment:
Why add this as an optional parameter to the method? Why not either introduce a new method that checks all signatures, or just make checking all signatures the default behaviour?

Boolean fields like this are, to me, generally indicative of a need for a new function as opposed to simply adding more configuration parameters and complexity to an existing one.